build-system/release-tagger/utils.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * GitHub API util functions.
+ * TODO: set owner repo defaults
+ * TODO: error handling
+ */
+
+// setup
+const {Octokit} = require('@octokit/rest');
```

Comment:
It took me an embarrassingly long time to realize that octokit is a cross between a cute octopus and a cute cat. So many [versions](https://www.google.com/search?q=github+octopus+cat&tbm=isch) of it!